mapeditor/windownewmap.cpp

```diff
@@ -202,7 +202,7 @@ std::unique_ptr<CMap> generateEmptyMap(CMapGenOptions & options)
 	map->initTerrain();
 	map->getEditManager()->clearTerrain(&CRandomGenerator::getDefault());
 
-	return std::move(map);
+	return map;
```

Comment:
Could you please please test that it works?

Reply:
Yes. Return'ing variable already allows move-constructing. Explicit std::move in return is actually detrimental to potential copy-elision.